src/main/java/com/researchspace/webapp/filter/SAMLRemoteUserPolicy.java

```diff
@@ -40,71 +34,57 @@ public class SAMLRemoteUserPolicy implements RemoteUserRetrievalPolicy {
   @Override
   public String getRemoteUser(HttpServletRequest httpRequest) {
 
-    log.debug("Req headers: {}", logHeaders(httpRequest).toString());
-    log.debug("Environment variables: {}", logEnv().toString());
-    log.debug("Attributes: {}", logReqAttributes(httpRequest).toString());
-    log.debug("SAML attributes: {}", logSamlAttributes(httpRequest).toString());
-    log.debug("Remote user:  {}", httpRequest.getRemoteUser());
+    SECURITY_LOG.debug("Req headers: {}", logHeaders(httpRequest));
+    SECURITY_LOG.debug("Environment variables: {}", logEnv());
+    SECURITY_LOG.debug("Attributes: {}", logReqAttributes(httpRequest));
+    SECURITY_LOG.debug("SAML attributes: {}", logSamlAttributes(httpRequest));
+    SECURITY_LOG.debug("Remote user:  {}", httpRequest.getRemoteUser());
```

Comment:
Do we still need all those debug lines? Are they possibly printing PI data?

Reply:
Yes these are occasionally useful and are not printed by default.